public/manifest/demo.json

```diff
@@ -24,5 +24,43 @@
       "display_name": "Sri Lanka",
       "isd_code": "94"
     }
+  ],
+  "v2": [
+    {
+      "country_code": "IN",
+      "display_name": "India",
+      "isd_code": "91",
+      "deployments": {
```

Comment:
We would need this to be an array to be extensible, right?

**Update:** Same comment for all non-prod manifests.

Reply:
Oops, thanks for catching 😅  Fixed in 6f4a443e4